CMakeLists.txt

```diff
@@ -31,15 +30,15 @@ target_link_libraries(range-v3 INTERFACE meta)
 
 add_subdirectory(doc)
 
-if(NOT RANGE_V3_NO_TESTING)
+if(RANGE_V3_TESTS)
   add_subdirectory(test)
 endif()
 
-if(NOT RANGE_V3_NO_EXAMPLE)
+if(RANGE_V3_EXAMPLES)
   add_subdirectory(example)
 endif()
 
-if(NOT RANGE_V3_NO_PERF)
+if(RANGE_V3_BENCHMARKS)
```

Comment:
`RANGE_V3_BENCHMARKS` may be overselling `perf`. (We probably shouldn't be shipping `perf` at all.) A more obscure name might steer people away and avoid wasting their time.

Reply:
Yes, I think that's an improvement.